src/server/letters-round/get-definition.js

```diff
@@ -0,0 +1,19 @@
+var fetch = require('isomorphic-fetch');
+
+function getDefinition(word) {
+    return fetch('https://glosbe.com/gapi/translate?from=eng&dest=eng&format=json&phrase=' + word + '&pretty=true')
+            .then(response => {
+                if (response.status >= 400) {
+                    throw new Error('Fakkin Error m9');
+                }
+                return response.json();
+            })
+            .then(response => {
+                return(response.tuc[0].meanings[0].text);
+            })
+            .catch(error => {
+                console.Error(error);
```

Comment:
You'll probably not want to catch the error here, rather in its usage so you can handle the error sensibly for the client
e.g. 
```
getDefinition(word).then(definition => {
 +        room.messageMembers('bestAnswer', {word: word.charAt(0).toUpperCase() + word.slice(1), definition: definition});    
 +    })
.catch(error => room.messageMembers('bestAnswer', {word: word.charAt(0).toUpperCase() + word.slice(1), error : "no def. avaible because: "  + error.reason}) );
```